core/src/main/codegen/templates/Parser.jj

```diff
@@ -4988,7 +5000,7 @@ SqlCall JsonExistsFunctionCall() :
 }
 {
     <JSON_EXISTS> { span = span(); }
-    <LPAREN> e = JsonApiCommonSyntax() {
+    <LPAREN> e = JsonApiCommonSyntax(true) {
```

Comment:
There is no need to set acceptNonPath = true for JsonApiCommonSyntax in functions other than JSON_LENGTH.